index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="AppBar">
+        <header class="ToolBar">
+          <nav>
+            <img src="images/NormalButton.png" alt="" class="NormalButton" />
+          </nav>
+          <div class="Logo">
+            <a href="#">faishar faike</a>
+          </div>
+        </header>
+      </div>
+      <div class="headline">
+        <div class="content">
+          <div class="PrimaryText">
+            <h1 class="PrimaryText">Hey There. I'm faishar</h1>
```

Comment:
Kindly use the Font-family, font-size and weight from the Figma design 🤗

![image](https://user-images.githubusercontent.com/73714615/192717137-d5b2c46c-aa55-4461-b0b7-f152a4b5a6e5.png)